### DISCUSSION TITLE
Revolutionising Telecom Microservice - Modernising JDiameter with Quarkus

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#42688] [Quarkus Blog/Website]
# Revolutionising Telecom Microservice - Modernising JDiameter with Quarkus

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-jdiameter-intro/

**Comment by jfrantzius** [comment]
Hi, the link to Quarkus JDiameter Documentation unfortunately is broken: https://docs.quarkiverse.io/quarkus-jdiameter/2.0.0/index.html

**Comment by holly-cummins** [comment]
Hm, yes, https://docs.quarkiverse.io/quarkus-jdiameter/dev/index.html seems to be the only valid link/version.

**Comment by holly-cummins** [comment]
I've raised https://github.com/quarkusio/quarkusio.github.io/pull/2126 (and also https://github.com/quarkusio/quarkusio.github.io/issues/2110), so closing. Thanks for reporting!
